### ISSUE TITLE
Rate Category on Scale - Android 

### ISSUE THREAD

**Issue by ghost** [whereToApplication/whereTo#65]
Rate Category on a scale (intense dislike, dislike, like, superlike).

**Comment by saahily** [comment]
I feel like this is an unnecessary feature.. don't want to burden the users too much

**Comment by ghost** [comment]
I agree that user burden is a concern. We spent some time brainstorming this out so let's talk about it more tomorrow @iecurius
